src/web/make-report.rkt

```diff
@@ -52,30 +52,26 @@
           ("Target" . "Color key:\nDark green: better than target\nGreen: matched target\nOrange: improved but did not match target\nYellow: no accuracy change\n")))
 
   (define total-time (apply + (map table-row-time tests)))
-  (define total-passed
-    (for/sum ([row tests])
-      (if (member (table-row-status row) '("gt-target" "eq-target" "imp-start")) 1 0)))
-  (define total-available
-    (for/sum ([row tests])
-      (if (not (equal? (table-row-status row) "ex-start")) 1 0)))
+  (define total-tests (length tests))
+  (define total-timeouts
+    (count (compose (curry equal? "timeout") table-row-status) tests))
   (define total-crashes
-    (for/sum ([row tests])
-      (if (equal? (table-row-status row) "crash") 1 0)))
-
-  (define total-gained
-    (for/sum ([row tests])
-      (or (table-row-result row) 0)))
+    (count (compose (curry equal? "crash") table-row-status) tests))
   (define total-start
-    (for/sum ([row tests])
-      (or (table-row-start row) 0)))
+    (for/sum ([t tests]) (or (table-row-start t) 0)))
+  (define total-result
+    (for/sum ([t tests]) (or (table-row-result t) 0)))
+  (define total-bits
+    (for/sum ([t tests])
+      ((compose
+        representation-total-bits
+        get-representation
+        table-row-precision)
+       t)))
 
   (define (round* x)
     (inexact->exact (round x)))
 
-  (define sorted-tests
-    (sort (map cons tests (range (length tests))) >
-          #:key (λ (x) (or (table-row-start (car x)) 0))))
-
```

Comment:
Do we not use this anywhere?

Reply:
It doesn't show uo anywhere else in that file, and it's scoped to `make-report-page`, so I assumed it wasn't used. Should it be though?

Reply:
Nah, I think we used to sort them but then we removed the sorting for some reason. Good to know we can just delete lines :)